TombEngine/Scripting/Internal/TEN/Misc/LogLevel.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <unordered_map>
+#include <string>
+
+/***
+Constants for LogLevel IDs.
+@enum Misc.LogLevel
+@pragma nostrip
+*/
+
+/*** Misc.LogLevel constants.
+
+The following constants are inside LogLevel.
+
+	WARNING
```

Comment:
As @hispidence mentioned later, you should add `ERROR` type in this comment as well.